### DISCUSSION TITLE
Zephyr SDK 0.14.0 does not contain a sysroots directory

### DISCUSSION THREAD

**Discussion by pdietl** [zephyrproject-rtos/zephyr#44387] [Q&A] [answered]
The Zephyr SDK 0.14.0 does not contain a `sysroots` directory in which the udev rule should be located per the getting started guide https://docs.zephyrproject.org/latest/getting_started/index.html#install-dependencies

**Comment by stephanosio** [comment, marked as answer]
You need to run the setup script (`setup.sh`) as described in the guide and install the host tools.
